spikeinterface/core/core_tools.py

```diff
@@ -8,6 +8,54 @@
 from .job_tools import ensure_chunk_size, ensure_n_jobs, divide_segment_into_chunks, ChunkRecordingExecutor
 
 
+def read_python(path):
```

Comment:
Why do we need this here ????
It was for PRB file only.
This function already in probeinterface  without `six`.

Reply:
`params.py` in Phy and klusta config file